scripts/install_ostis.sh

```diff
@@ -1,69 +1,74 @@
 #!/bin/bash
 
+APP_ROOT_PATH=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && cd .. && pwd)
+PLATFORM_PATH="${APP_ROOT_PATH}/ostis-web-platform"
+WORKING_PATH=$(pwd)
+PYTHON_PATH=${APP_ROOT_PATH}/problem-solver/py/services
 PLATFORM_REPO="https://github.com/ostis-dev/ostis-web-platform.git"
-PLATFORM_FOLDER="ostis-web-platform"
 
 prepare_platform()
 {
-	cd scripts
+	cd ${PLATFORM_PATH}/scripts
 	./prepare.sh
-	cd ..
+	cd ${PLATFORM_PATH}
 }
 
 prepare_platform_without_build()
 {
-	cd scripts
+	cd ${PLATFORM_PATH}/scripts
 	./prepare.sh no_build_kb no_build_sc_machine
-	cd ..
+	cd ${PLATFORM_PATH}
 }
 
 include_kb()
 {
+	cd ${PLATFORM_PATH}
 	rm ./ims.ostis.kb/ui/ui_start_sc_element.scs
 	rm -rf ./kb/menu
 	echo "../kb" >> ./repo.path
 	echo "../problem-solver" >> ./repo.path
 	cd scripts
 	./build_kb.sh
-	cd ..
+	cd ${PLATFORM_PATH}
 }
 
 include_kpm()
 {
-	cd sc-machine
+	cd ${PLATFORM_PATH}/sc-machine
 	sed -i '\|build/problem-solver/cxx|d' ./CMakeLists.txt
 	sed -i '\|build/problem-solver/py|d' ./CMakeLists.txt
-	cat ../../scripts/sc_machine_cmake_file_ext.txt >> ./CMakeLists.txt
+	cat "${APP_ROOT_PATH}"/scripts/sc_machine_cmake_file_ext.txt >> ./CMakeLists.txt
 	
-	cd config
-	sed -i '/python/d' ./config.ini.in
-	sed -i '/modules_path/d' ./config.ini.in 
-	echo -e '[python]' >> ./config.ini.in
-	echo 'modules_path = ${SC_MACHINE_ROOT}/sc-kpm/sc-python/services;${SC_MACHINE_ROOT}/../../problem-solver/py/services' >> ./config.ini.in
-	cd ..
-	
-	cd ./scripts
+	if ! grep -q "${PYTHON_PATH}" "${PLATFORM_PATH}/sc-machine/config/config.ini.in";
+	then
+  	PYTHON_PATH_ESCAPED="$(echo "${PYTHON_PATH}" | sed -e 's/[$/]/\\&/g')"
+  	sed -i "/modules_path/ s/$/;${PYTHON_PATH_ESCAPED}/" "${PLATFORM_PATH}/sc-machine/config/config.ini.in"
+	fi
+
+	cd scripts
 	./make_all.sh
-	cat ../bin/config.ini >> ../../config/sc-web.ini
-	cd ../..
+	cat ../bin/config.ini >> "${PLATFORM_PATH}"/config/sc-web.ini
```

Comment:
'..' is replaced everywhere but here. Shouldn't it be `cat "${PLATFORM_PATH}"/sc-machine/bin/config.ini`?